jffs2/phoenix-rtos/mtd.c

```diff
@@ -39,25 +39,26 @@ struct dentry *mount_mtd(struct file_system_type *fs_type, int flags,
 
 	/* Initialize MTD */
 	if (p->strg->dev->mtd->type == mtd_norFlash) {
-		mtd->name = p->strg->dev->mtd->name;
 		mtd->type = MTD_NORFLASH;
-		mtd->erasesize = p->strg->dev->mtd->erasesz;
-		mtd->writesize = p->strg->dev->mtd->writesz;
-		mtd->flags = MTD_WRITEABLE;
-		mtd->size = p->strg->size;
-
-		mtd->index = 0;
-		mtd->oobsize = 0;
-		mtd->oobavail = 0;
-		mtd->storage = p->strg;
 	}
 	else if (p->strg->dev->mtd->type == mtd_nandFlash) {
-		/* TODO */
+		mtd->type = MTD_NANDFLASH;
```

Comment:
Hmm, I guess this `mtd_nandFlash` is our own const and `MTD_NANDFLASH` is a standard one? If so it would be better to remove `mtd_nandFlash` and use standard one

Reply:
In this case `mtd_` prefix is missleading - this enum does not apply to mtd module (libmtd), but to storage. So `storage_` prefix is more appropriate